controllers/admin/AdminStatsController.php

```diff
@@ -427,36 +522,80 @@ public static function getPurchases($date_from, $date_to, $granularity = false)
     {
         if ($granularity == 'day') {
             $purchases = array();
-            $result = Db::getInstance(_PS_USE_SQL_SLAVE_)->ExecuteS('
+            $resultOrders = Db::getInstance(_PS_USE_SQL_SLAVE_)->ExecuteS('
 			SELECT
-				LEFT(`date_upd`, 10) as date,
-				SUM(od.`product_quantity` * IF(
+				LEFT(o.date_upd, 10) as date,
+				SUM((od.product_quantity - coalesce(sd.product_quantity,0) * IF(
```

Comment:
There's a missing closing parenthesis in `(od.product_quantity - coalesce(sd.product_quantity,0)`